boards/px4/fmu-v5/default.px4board

```diff
@@ -6,6 +6,8 @@ CONFIG_BOARD_SERIAL_TEL2="/dev/ttyS2"
 CONFIG_BOARD_SERIAL_TEL4="/dev/ttyS3"
 CONFIG_DRIVERS_ADC_ADS1115=y
 CONFIG_DRIVERS_ADC_BOARD_ADC=y
+CONFIG_ATMOSPHERIC_QUANTITIES=y
+CONFIG_DRIVERS_ATMOSPHERIC_QUANTITIES_SHT3x=y
```

Comment:
Doesn't ATMOSPHERIC_QUANTITIES feel like a bit of an over generalization given we separately have barometers, etc?

How about just using `hygrometer` as the category/directory for now?

```suggestion
CONFIG_DRIVERS_HYGROMETER_SHT3x=y
```

Reply:
We want to create more atmospheric sensors in the near future. Hence this name. Therefore, I think this name is appropriate. And I'd like to keep it under the same folder/group

Reply:
You don't think it overlaps with existing categories like barometric pressure (barometer)? The category seems too broad to me.

Reply:
Hi @dagar, I renamed it to `hygrometer`. Yes, it's more specific

Reply:
Thanks, we can always reorganize these things in the future as more options or better ideas come along.